### DISCUSSION TITLE
Welcome to ecoevo_1000 Discussions!

### DISCUSSION THREAD

**Discussion by daniel1noble** [daniel1noble/ecoevo_1000#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by itchyshin** [comment]
@daniel1noble - thanks for this. I am just testing this Discussion stuff - my first time. 

I am Shinichi Nakagawa (@itchyshin) and a co-organiser of this hackathon. I am also a cofounder of EcoEvoRxiv.

**Comment by mlagisz** [comment]
@daniel1noble - thanks! It is my first time using GitHub Discussion too.
I am Losia (Malgorzata) Lagisz (@mlagisz), also a hackathon co-organiser. I was helping with EcoEvoRxiv info webpages in the past, but now mostly submitting preprints! Great to be here!

**Comment by daniel1noble** [comment]
I'm Daniel Noble (@daniel1noble) currently the Chair of the EcoEvoRxiv preprint server and one of its founding members along with my wonderful Hackathon co-organisers!

**Comment by shafieisabets** [comment]
Hi everyone! My name is Saeed Shafiei Sabet, a Behavioural biologist and reader in animal behaviour ([@SaeedSHSABET](https://twitter.com/SaeedSHSABET))  and https://github.com/shafieisabets @shafieisabets

**Comment by elviradbastiani** [comment]
Hi all! I am Elvira D'Bastiani from Brazil, but currently working at UCLA Los Angeles, US. I'm interested in the ecological and evolutionary dynamics of host-parasites. Now I am investigating how spatiotemporal scales and social interactions influence disease transmission. At EcoEvoRxiv, I am helping as an editor of English and Portuguese. Excited to meet new people and to see all the participants!!
